tests/test_huawei.py

```
@@ -0,0 +1,392 @@
+    assert "qr_code" in wifi["guest"]
+    assert wifi["main_5g"]["on"] == 1
+    assert wifi["guest_5g"]["ssid"].endswith("Guest")
+
+
+def test_set_wifi_status_posts_enable(
+    base_url: URL, monkeypatch: pytest.MonkeyPatch
+) -> None:
+    """set_wifi_status posts Enable to the correct WLANConfiguration instance."""
+    api = _api(base_url)
+    api._session_cookie = "Cookie=sid=test"  # noqa: SLF001
+    api.csrf_token = "tok123"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "csrf_token"

(S105)